criterion.cabal

```diff
@@ -111,27 +111,16 @@ library
   else
     ghc-options: -O2
 
--- [2016.05.30] RRN: I'm not sure where this was going.
--- It looks like it was meant for post-facto analysis of
--- reports?  Will have to ask BOS.
---------------------------------------------------------
--- executable criterion
---   hs-source-dirs: app
---   main-is:        App.hs
---   other-modules:
---     Options
```

Comment:
The Travis build appears to be failing because you removed `other-modules: Options`.